.github/workflows/ci.yml

```diff
@@ -0,0 +1,214 @@
+#  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change to 2021

Reply:
Resolved in https://github.com/asgardeo/asgardeo-dotnet-oidc-sdk/pull/24/commits/db9ac3dd97e72776a648ac0090c043ca0dd5823b